app/controllers/support_interface/ucas_matches_controller.rb

```diff
@@ -15,26 +15,40 @@ def audit
       @match = UCASMatch.find(params[:id])
     end
 
-    def process_match
+    def record_initial_emails_sent
       match = UCASMatch.find(params[:id])
-      match.update!(matching_state: :processed)
-      flash[:success] = 'Match marked as processed'
+      match.update!(
+        candidate_last_contacted_at: Time.zone.now,
+        action_taken: 'initial_emails_sent',
+      )
+      flash[:success] = 'The date of the initial emails was recorded'
       redirect_to support_interface_ucas_match_path(match)
     end
 
-    def record_initial_emails_sent
+    def record_reminder_emails_sent
       match = UCASMatch.find(params[:id])
-
-      if match.update!(
-        action_taken: 'initial_emails_sent',
+      match.update!(
         candidate_last_contacted_at: Time.zone.now,
+        action_taken: 'reminder_emails_sent',
       )
-        flash[:success] = 'The date of the initial emails was recorded'
-      else
-        flash[:warning] = 'There was a problem and the date of the initial emails was not recorded'
+      flash[:success] = 'The date of the reminder emails was recorded'
+      redirect_to support_interface_ucas_match_path(match)
+    end
```

Comment:
@AgaDufrat the card specifies that the `ChaserSent` model should be used when reminder emails are sent, so I think this should take place here, and record the interaction.

Reply:
Sending emails automatically will be done in the next iteration. Sorry that the card is not accurate!
At the moment support agents send all the emails via Zendesk and only confirm/record that they performed the action by clicking the buttons in Apply service.

Reply:
👍